### PR TITLE
add installing via composer

### PR DESCRIPTION
**Pull request by AhmedHelalAhmed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ in our [Gitter](https://gitter.im/ruflin/Elastica) channel.
 All library issues should go to the [issue tracker from github](https://github.com/ruflin/Elastica/issues).
 
 
+Installing
+-------------
+composer require ruflin/Elastica
```
[the remark this answers is not in the document]
Comment (AhmedHelalAhmed) [reply README.md]: As a developer, I would like to use command to install it.

[the remark this answers is not in the document]
Comment (ruflin) [reply README.md]: I wonder if we should rather update the docs here: http://elastica.io/getting-started/installation.html ?

[the remark this answers is not in the document]
Comment (AhmedHelalAhmed) [reply README.md]: I think yes, it's better to make the installation with a command

[the remark this answers is not in the document]
Comment (ruflin) [reply README.md]: I would not judge if one is better over the other but we should have both there. Do you want to open a PR agains https://github.com/ruflin/elastica.io ?

**Comment by deguif** [comment]
Closing as old and conflicting.

**Pull request closed without merging**